packages/pg-pubsub/__tests__/subs.test.ts

```diff
@@ -15,8 +17,21 @@ import { runQuery, TestCtx } from "./runQuery";
 let ctx: TestCtx | null = null;
 const CLI_DEFAULTS = {};
 
-const printIntrospectionResults = (json: any) =>
-  printSchema(buildClientSchema(json.data));
+const GraphQLSchemaSerializer = {
+  test(val: any) {
+    return isSchema(val);
+  },
+  // It would be nice if we could name this method `serialize`,
+  // so that it conforms to the shape of `pretty-format.NewPlugin`,
+  // but the Jest types don't seem to know about that type yet,
```

Comment:
From looking at the source, it looks like they should as it's declared inside pretty-format's `.d.ts` files here:

https://unpkg.com/browse/pretty-format@24.9.0/build/types.d.ts

Is our version of pretty-format out of date? If so, try deleting it's definition in `yarn.lock` and running yarn again to get the latest. If not I'm happy to merge as-is and research more later.

Reply:
As far as I can tell, this is happening because we depend on `@types/jest`, which is not kept perfectly in sync with the actual types in Jest. As far as I can tell, `@types/jest` was created because Jest was originally written in Flow. However, now Jest has migrated to Typescript, but `@types/jest` still exists -- and is apparently still necessary, for some reason. I tried removing `@types/jest` from our `package.json` files and then running `yarn` to uninstall it, but the tests failed to run. I got the following error:

```
__tests__/index.test.ts:6:1 - error TS2582: Cannot find name 'test'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
```

Ironic, huh?

Reply:
This issue seems relevant: https://github.com/facebook/jest/issues/8794

Reply:
Noted. Merge as-is then :)